agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -172,9 +174,25 @@
 
     public static class DestinationFile {
 
-        public String path = "applicationinsights.log"; // relative to the directory where agent jar is located
+        private static final String DEFAULT_NAME = "applicationinsights.json";
+
+        public String path = getDefaultPath();
         public int maxSizeMb = 5;
         public int maxHistory = 1;
+
+        private static String getDefaultPath() {
+            if (!DiagnosticsHelper.isRpIntegration()) {
+                // this will be relative to the directory where agent jar is located
+                return DEFAULT_NAME;
+            }
+            if (DiagnosticsHelper.useAppSvcRpIntegrationLogging()) {
```

Comment:
For sping cloud, which if statement does it fall through?

Reply:
oh my, good point, fixed!